css/responsive/desktop.styl

```diff
@@ -6,6 +6,7 @@ $gutter = 32px
   
 .col-l
   padding: 0 ($gutter / 2)
+  margin-bottom: ($gutter / 2)
```

Comment:
yes but it can be avoided. every column item needs the correct margins and paddings on every grid size so you can just put these rules on `.col` and avoid extra markup